cli-mule/src/main/ballerina/tool-bi-migrate-mule/README.md

```diff
@@ -91,8 +122,9 @@ project.
     2. **Overall Project Conversion Percentage** – Represents the combined conversion rate based on both component-level
 and DataWeave conversions.
 
-- A detailed report is generated as `migration_summary.html` in the root of the newly created Ballerina package. This
-  report provides a breakdown of the conversion percentage.
+- A detailed report is generated as `migration_summary.html` in the root of the newly created Ballerina package. 
+  This report provides the percentage of automated migration coverage and an estimated time for completing the remaining parts.
+  Sections that require manual conversion are also highlighted in the report.
```

Comment:
Is this line break (at Sections) intentional. If so I don't think markdown renders the linebreak (You need an empty line between two)

Reply:
This is because my IDE automatically go to next line after 120. Don't excepting a line break in the rendering